build/artifacts/rhel.Dockerfile

```diff
@@ -10,8 +10,8 @@
 #
 
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/devtools/go-toolset-rhel7
-FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.12.12-4 as builder
-ENV PATH=/opt/rh/go-toolset-1.12/root/usr/bin:$PATH \
+FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.13.4-18 as builder
+ENV PATH=/opt/rh/go-toolset-1.13/root/usr/bin:$PATH \
```

Comment:
I wonder if it makes sense to keep Dockerfile and rhel.Dockerfile sync.
Seems Dockefile still use 1.12 go